incubator/hnc/pkg/kubectl/root.go

```diff
@@ -43,6 +44,7 @@ type realClient struct{}
 type Client interface {
 	getHierarchy(nnm string) *api.HierarchyConfiguration
 	updateHierarchy(hier *api.HierarchyConfiguration, reason string)
+	hasNamespace(nnm string) bool
```

Comment:
This is never called through the `Client` interface, so it doesn't need to be here. Just having it on `realClient` struct is enough.